pkg/ai/bedrock.go

```diff
@@ -0,0 +1,35 @@
+package ai
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/service/bedrockagentruntime"
+	"github.com/aws/aws-sdk-go-v2/service/bedrockagentruntime/types"
+)
+
+// InvokeAWSBedrockAgent invokes the AWS Bedrock agent with the provided parameters.
+func InvokeAWSBedrockAgent(cfg aws.Config, ctx context.Context, agentID, inputText, sessionID string) (string, error) {
```

Comment:
this is missing the prompt from AWS, right? is it assuming we will just hardcode them?

Reply:
I didn't see an option in the SDK for referencing a prompt, only the agent. The prompt would be more ideal, but if we can't use that, yes, hardcoding is probably the way to go.